### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by StevenSun1234**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by StevenSun1234 — COMMENTED** [review]
Should be ok considering this is an exact template of the lesson

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (stevejoe1432) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (stevejoe1432) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
 <script>
```
Comment (stevejoe1432) [inline index.html:52]:
```suggestion
<script>
```

File: index.html (hunk 1 above)
Comment (StevenSun1234) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again, man 🧑‍💻 🤖!</h2>
```

**Pull request merged**